### DISCUSSION TITLE
Onion Skinning

### DISCUSSION THREAD

**Discussion by xItzDiamondx** [ScratchAddons/ScratchAddons#2111] [Ideas] [answered]
Hi. It's me again. So I was just animating on Scratch and I thought, "You know, onion skinning would be useful" So, I wanted to ask y'all if you can make this an addon in the extension. Also, please include it in Turbowarp. (If you can.)

**Comment by xItzDiamondx** [comment]
Also, I will gladly help develop it.

**Comment by retronbv** [comment, marked as answer]
Onion Skinning is already an addon

**Reply by xItzDiamondx** [reply]
It is?

**Reply by xItzDiamondx** [reply]
Also, is there anything I can help develop then. And how can I include it in TurboWarp?

**Reply by TheColaber** [reply]
Just enable it at https://turbowarp.org/addons

**Reply by xItzDiamondx** [reply]
Hmmm.. Doesn't seem to work

**Reply by TheColaber** [reply]
![image](https://user-images.githubusercontent.com/72760579/114192147-26db3900-991b-11eb-9c10-1a1e7e1ab613.png)

**Comment by cobaltt7** [comment]
What version are you on?

**Reply by xItzDiamondx** [reply]
What do you mean

**Comment by WorldLanguages** [comment]
Is this a troll? :P

**Reply by retronbv** [reply]
I hope it isn’t
